javascript/builtins/Object.json

```diff
@@ -2126,7 +2128,8 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": "1"
+                "version_added": "1",
+                "notes": "Starting in Firefox 74, <code>toString()</code> is no longer available for use by web content. It is still allowed for internal and privileged code."
```

Comment:
`toString` is removed? I don't believe that :)
 I think you meant this change for `toSource`?

Reply:
Gah. Yeah. Brain damage, thanks for the catch. And I had assumed that objects were falling back to Object for the toSource() doc, which was a mistake to do so (it would have been valid in the interface docs, which is where I usually work these day). I will go on the hunt. Thanks.